pkg/util/winutil/users_test.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018-present Datadog, Inc.
+
+// +build windows
+
+package winutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetSidFromUser(t *testing.T) {
+	sid, err := GetSidFromUser()
+	t.Logf("The SID found was: %v", sid)
+	assert.Nil(t, err)
+}
+
+func TestGetUserFromSid(t *testing.T) {
+	sid, err := GetSidFromUser()
+	assert.Nil(t, err)
```

Comment:
Some assertions around `sid` value would be good too

Reply:
So, to assert this, I would need to either call the same code that is being called within the function, which doesn't make sense. Or, I would have to make that function mockable, which I could do but the test then loses value as a test because of all the chained calls to the `windows` API.

Reply:
I'll just assert it's `NotNil`.